index.html

```diff
@@ -40,32 +41,277 @@ <h1 class="main-header">I’m Suliman Joya Glad to see you!</h1>
           <ul class="social-icons">
             <li class="icon">
               <a href="#">
-                <img src="images/twitter.svg" alt="twitter logo">
+                <img src="images/twitter.svg" alt="twitter logo" />
               </a>
             </li>
             <li class="icon">
               <a href="#">
-                <img src="images/Linkedin-icon.svg" alt="Linkedin logo">
+                <img src="images/Linkedin-icon.svg" alt="Linkedin logo" />
               </a>
             </li>
             <li class="icon">
               <a href="#">
-                <img src="images/M.svg" alt="M logo">
+                <img src="images/M.svg" alt="M logo" />
               </a>
             </li>
             <li class="icon">
               <a href="#">
-                <img src="images/Github.svg" alt="Github logo">
+                <img src="images/Github.svg" alt="Github logo" />
               </a>
             </li>
             <li class="icon">
               <a href="#">
-                <img src="images/Peace.svg" alt="peace logo">
+                <img src="images/Peace.svg" alt="peace logo" />
               </a>
             </li>
           </ul>
         </main>
       </div>
+      <!-- The intro page is end here -->
+      <!-- ************************************ -->
+      <!-- The project pages are start here -->
+      <div class="projects-page">
+        <div class="all-projects">
```

Comment:
I suggest you use a `section` tag here. `div` is too general. Its not only good for semantic purposes but for screen readers too.